test/config-next/nonce.json

```diff
@@ -1,7 +1,8 @@
 {
   "NonceService": {
     "maxUsed": 131072,
-    "noncePrefix": "taro",
+    "useDerivablePrefix": true,
+    "prefixSalt": {"passwordFile": "test/secrets/nonce_salt"},
```

Comment:
Are the contents of this file supposed to be unique to for each instance of the nonce server? Does a single wfe2 redeem a single nonce server salt at first? What happens if I have multiple nonce servers and multiple wfe2?

Reply:
They should actually be same across all of these instances. See the comments for these fields in the `nonce-service` and `boulder-wfe` configuration fields.